dask/dataframe/core.py

```diff
@@ -1462,6 +1462,18 @@ def append(self, other):
             other = other.to_frame().T
         return super(DataFrame, self).append(other)
 
+    @derived_from(pd.DataFrame)
+    def iterrows(self):
+        for i in range(self.npartitions):
+            for row in self._default_get(self.dask, (self._name, i)).iterrows():
+                yield row
+
+    @derived_from(pd.DataFrame)
+    def itertuples(self):
+        for i in range(self.npartitions):
+            for row in self._default_get(self.dask, (self._name, i)).itertuples():
+                yield row
```

Comment:
This is somewhat important.  We probably shouldn't use `_default_get` internally because it doesn't go through the  normal optimization paths.  The suggestion above should work well.